src/app/modules/main/components/welcome/welcome.component.scss

```diff
@@ -1,47 +1,86 @@
-@import "../../../../core/styles/plugins/variables";
+@import "plugins/variables";
+@import "plugins/mixins";
 
 .welcome {
-  background-image: url("../../../../../assets/welcome-background.gif");
-  background-repeat: no-repeat;
-  background-size: cover;
-  background-position: center center;
-  background-attachment: fixed;
-  top: 0;
-  margin: 0;
-  padding: 0;
-  width: 100%;
-  height: 100vh;
+  min-height: 100vh;
+  position: relative;
+  background: url(/assets/backgrounds/welcome-background.gif) no-repeat center top / cover;
+  transform: scaleX(-1);
+  padding: 4%;
+  box-sizing: border-box;
 
   &__container {
-    margin: 12% 23% 12% 43%;
+    transform: scaleX(-1);
+    align-self: stretch;
+  }
+
+  &__heading {
+    @include text-stroke();
+    max-width: 30%;
+    margin: 0 auto;
     text-align: center;
+    font-size: 2em;
+    font-weight: $bold;
+    font-family: $ff-header-solid;
+    color: $tertiary-color;
+    text-transform: uppercase;
+  }
+
+  &__game {
+    max-width: 30%;
+  }
+}
+
+.game {
+  background: url(/assets/images/shirt.png/) no-repeat left top / contain;
+  padding: 6% 8%;
+
+  &__heading {
+    text-align: center;
+    margin-bottom: 1em;
   }
 
   &__title {
-    font-size: 2rem;
-    text-transform: uppercase;
-    font-weight: bold;
-    padding: .5rem;
+    font-family: $ff-header-solid;
+    font-size: 2em;
+    @include text-stroke();
+    color: $primary-color;
   }
 
-  &__name {
-    font-size: 3rem;
-    // color: #00a86b;
+  &__description {
+    font-family: $ff-secondary;
+    margin-bottom: 2em;
   }
 
-  &__game-button {
-    padding: .5rem 1rem;
-    margin: 0.5rem auto;
-    width: 100px;
-    border-radius: 10px;
-    background: $secondary-color;
-    font-size: 1.2rem;
-    text-transform: uppercase;
-    font-weight: bold;
-    font-family: "Raleway", sans-serif;
+  &__buttons {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: row wrap;
+  }
+
+  &__button {
+    font-size: 1.3em;
+    border: 2px solid #000000;
+    letter-spacing: 0.1em;
 
-    &:hover {
-      background-color: $primary-color;
+    &:first-child {
```

Comment:
Better to don't use selector's like first-child, nth-child etc. Because if element's will be swap, style gonna work wrong. Use classes instead it's better, as for me